tests/UnitTests/Rsp.IrasPortal.UnitTests/Services/ReviewBodyServiceTests/CreateReviewBodyTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Rsp.IrasPortal.Application.DTOs;
+using Rsp.IrasPortal.Application.Responses;
+using Rsp.IrasPortal.Application.ServiceClients;
+using Rsp.IrasPortal.Services;
+
+namespace Rsp.IrasPortal.UnitTests.Services.ReviewBodyServiceTests;
+
+public class CreateReviewBodyTests : TestServiceBase<ReviewBodyService>
+{
+    [Theory]
+    [AutoData]
+    public async Task CreateReviewBody_Should_Return_Failure_Response_When_Client_Returns_Failure(
+        ReviewBodyDto reviewBodyDto)
+    {
+        // Arrange
+        var apiResponse = Mock.Of<IApiResponse>(
+            apiResponse => !apiResponse.IsSuccessStatusCode &&
+                           apiResponse.StatusCode == HttpStatusCode.BadRequest);
+
+        var client = new Mock<IReviewBodyServiceClient>();
+        client.Setup(c => c.CreateReviewBody(reviewBodyDto))
+            .ReturnsAsync(apiResponse);
+
```

Comment:
please move the Setup and Return calls on a separate line for better readability. Same issue quite a few of the other tests across the project.

```suggestion
        client
            .Setup(c => c.CreateReviewBody(reviewBodyDto))
            .ReturnsAsync(apiResponse);
```